format/FormatHDF5Exceptions.py

```diff
@@ -0,0 +1,36 @@
+import h5py
+
+# static enumerations for useless file types
+
+from enum import Enum, auto
+class HDF5_FileType(Enum):
+    DATA=auto()
+    META=auto()
+    MPCCD=auto()
+    NXS=auto()
+    UNKNOWN=auto()
```

Comment:
I just found out that `Enum` allows `namedtuple`-style invocation which is much cleaner:
```suggestion
from enum import Enum
HDF5_FileType = Enum("HDF5_FileType", "DATA META MPCCD NXS UNKNOWN")
```